UnitTests/Tests/TRWeatherUpdateCacheSpec.m

```diff
@@ -29,6 +29,14 @@ - (NSURL *)latestWeatherUpdateURL;
     [[NSFileManager defaultManager] removeItemAtURL:weatherUpdateURLForTesting() error:nil];
 };
 
+NSDictionary* (^weatherConditionsWithTemperature) (NSNumber*) = ^NSDictionary* (NSNumber *temp) {
```

Comment:
I would like not doing this. `TRWeatherUpdateCacheSpec` will be removed in next pull request and replaced with `WeatherUpdateCacheSpec` written in Swift.